developer_guide.md

```diff
@@ -111,10 +117,15 @@ Run the following command to get the javascript code recompiled when changed:
 
 Python code change will trigger a restart of the Flask server automatically.
 
-### Disable Mixer branch cache
+### Disabled Mixer branch cache
```

Comment:
thanks for the update! this section can be removed now. The branch cache is read from BT and does not impact startup time or mem usage

Reply:
ok! removed